packages/smart-wallet/src/offers.js

```diff
@@ -180,7 +180,11 @@ export const makeOfferExecutor = ({
           handleError,
         );
       };
-      await tryBody().catch(err => handleError(err));
+      await tryBody().catch(err => {
+        handleError(err);
+        // propagate to caller
```

Comment:
If the caller is a bridge handler, I'm not sure throwing is allowed. Is there something higher up that deals with that?

Reply:
PTAL 71314fe06efa35927373cee662491805345a93b4